docs/tutorials/03-publishing-pilets.md

```diff
@@ -19,52 +19,121 @@ We designed Piral to be as simple as possible, while remaining as flexible and p
 
-**Remark:** The specification for a feed service is public. There should be everything you need if you already (or later) want to start building your own feed service. In addition to the publicly available specification we also have published a Node.js sample implementation at GitHub (https://github.com/smapiot/sample-pilet-service). The sample implementation can also be installed or run locally very easily.
+**Remark:** The specification for a feed service is public. There should be everything you need if you already (or later) want to start building your own feed service. In addition to the publicly available specification we also have published a Node.js sample implementation at GitHub (<https://github.com/smapiot/sample-pilet-service>). The sample implementation can also be installed or run locally very easily.
 
-The service can be found at https://feed.piral.io. Registration is possible by using a Microsoft Account to log into the service. We do not need any personal data besides the email / account id, which is provided after your approval.
+The service can be found at <https://feed.piral.io>. Registration is possible by using a Microsoft Account to log into the service. We do not need any personal data besides the email / account id, which is provided after your approval.
 
-Logging into the service you'll see the overview page. At this point nothing will be displayed as you have not created any feed yet.
+For our tutorials, we will utilize this publicly available community edition of the feed service.
 
 [^1]: For development purposes and usage. The exact limitations are subject to change. Please see details on our feed service website.
 
-## Creating a Feed
+## Overview
+In the Getting Started tutorial, we covered already the creation of a Piral instance and its first pilet. This tutorial will guide us through the steps how to publish our first pilet to the feed service and fetch it subsequently into the Piral instance.
 
-For publishing (or retrieving) pilets you will need a feed. A feed is like a container or folder. The link for creating the feed service can be found in the top menu.
+This quick start will show us how to
 
-![Feed Service Header](../diagrams/feed-service.png)
+1. Creating a **Feed** in the feed service
+2. Create an **API Key** required for publishing the pilet
+2. **Publish a pilet** to the feed service
+3. **Configure the Piral Instance** to fetch pilets from the feed service
 
-To create the feed we need to give it a proper (unique) name. The system will check if the provided name is still available. The description can only be seen by you - it is optional. The allowed hosts you can also leave blank for now (resolving to `*`).
+## Prerequisites
 
-When you are done just press "create" and wait for the operation to complete. You will now be feed details page, which shows you the available pilets.
+For completing the tutorials, the following prerequisites must be met:
```

Comment:
This is redundant because the premise of the tutorials is that they have to be done in order. If we break this we would need also to explain things such as "what is a pilet" or "how to scaffold a Piral instance".

Here we can just write - we assume that a Piral instance and a pilet is available. If not, please follow the getting started tutorial.

Reply:
Updated as suggested.